openedx/stanford/cms/djangoapps/contentstore/tests/test_bulkupdate.py

```diff
@@ -0,0 +1,94 @@
+"""
+Unit tests for bulk update problem settings utility
+
+TODO Write unit tests for bulk_update_problem_settings from utilities.tasks
+"""
+
+import ddt
+
+from xmodule.capa_base import CapaFields
+from xmodule.modulestore.tests.factories import CourseFactory
+
+from cms.djangoapps.contentstore.tests.utils import CourseTestCase
+from contentstore.utils import reverse_course_url
+
+from ..views.utilities.bulkupdate import SHOW_ANSWER_OPTIONS
+
+
+SHOW_ANSWER_OPTIONS = []
```

Comment:
We shouldn't be rebuilding this here,
since we're importing it already, right?

Reply:
Forgot to take the import out, but now that it's brought up...for tests should we be rebuilding the list or importing from the file being tested?